cypress/integration/Navigation.test.js

```diff
@@ -7,21 +7,43 @@ function getBtnAndClick(selector, content) {
     .click();
 }
 
+function fillFormInput(name, content) {
+  return cy.get(`[data-testid="delivery-form"] input[name=${name}]`).type(content);
```

Comment:
I think **data-testid** is not required here, since in most cases you want to have only 1 form per page.
Probably `form input[name=${name}]` will be sufficient.